manimlib/camera/three_d_camera.py

```diff
@@ -218,15 +365,45 @@ def get_static_center_func(mobject):
                 self.fixed_orientation_mobjects[submob] = func
 
     def add_fixed_in_frame_mobjects(self, *mobjects):
+        """This method allows the mobject to have a fixed position,
+        even when the camera moves around.
+        E.G If it was passed through this method, at the top of the frame, it
```

Comment:
nit: e.g. should have a . after the g and the two letters needn't both be capitalized.